react-frontend/src/shared-components/peerly-logo/LogoComponent.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Styled from "styled-components";
+
+import LabelComponent from "core-components/label/LabelComponent";
+import Row from "core-components/row/RowComponent";
+import Col from "core-components/col/ColComponent";
+
+const Logo = Styled.div`
+& {
+  width: 50px;
+  height: 50px;   
+  position: absolute;
+  top: 10px;
+  transform: rotate(45deg);
+}
+&: before {
+  content: "";
+  width: 20px;
+  height: 20px;
+  border: 2px solid #FFFFFF;              
+  position: absolute;
+  transform: translate(-50%,-50%);
+  top: 50%;
+  left: 50%;
+}  
+`;
+const PlusSign = Styled.div`
+position: absolute;
+top: 55%;
```

Comment:
indentation fix, when we add a single comment, always check if the same mistake is repeated elsewhere